.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines": {"/Users/ericdavis/chapter_3/sywtltc_chapter_3_1/test_calculator.py": [1, 2, 4, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 18, 19, 20, 21, 22, 24, 25, 26, 27], "/Users/ericdavis/chapter_3/sywtltc_chapter_3_1/calculator.py": [1, 3, 4, 5, 6, 7, 8, 9]}}
```

Comment:
only add your py and test files, don't add a lot of these support files. they end up just bloating the repo and making peer reviews harder.